modules/fileout_covjson/FoDapCovJsonTransform.cc

```diff
@@ -1648,6 +1662,80 @@ cout <<"grid map name: "<<(*i)->name() <<endl;
     printCoverageJSON(strm, indent, testOverride);
 }
 
+void FoDapCovJsonTransform::transform(ostream *strm, libdap::DMR *dmr, const string& indent, bool sendData, bool testOverride)
+{
+
+#if 0
```

Comment:
If you want to keep this because it's needed, that's OK, but otherwise it will be easier if it is removed.

Reply:
Will remove this  #if 0  block.